src/Microsoft.PowerShell.ConsoleHost/WindowsTaskbarJumpList/TaskbarJumpList.cs

```diff
@@ -4,14 +4,40 @@
 using System;
 using System.Diagnostics;
 using System.Reflection;
+using System.Threading;
 
 using static Microsoft.PowerShell.ComInterfaces;
 
 namespace Microsoft.PowerShell
 {
     internal static class TaskbarJumpList
     {
-        internal static void CreateElevatedEntry(string title)
+        // Creating a JumpList entry takes around 55ms when the PowerShell process is interactive and
+        // owns the current window (otherwise it does a fast exit anyway). Since there is no 'GET' like API,
+        // we always have to execute this call because we do not know if it has been created yet.
+        // The JumpList does persist as long as the filepath of the executable does not change but there
+        // could be disruptions to it like e.g. the bi-annual Windows update, we decided to
+        // not over-optimize this and always create the JumpList as a non-blocking background task instead.
+        internal static void CreateRunAsAdministratorJumpList()
+        {
+            // Some COM APIs are implicitly STA only, therefore the executing thread must run in STA.
+            var thread = new Thread(() =>
+            {
+                try
+                {
+                    TaskbarJumpList.CreateElevatedEntry(ConsoleHostStrings.RunAsAdministrator);
+                }
+                catch
+                {
+                    // Due to COM threading complexity there might still be sporadic failures but they can be
+                    // ignored as creating the JumpList is not critical and persists after its first creation.
```

Comment:
If don't write to a log can put at least Debug.Assert() here?

Reply:
Ok, done now.